planetarium/NineChronicles
# Show popup on crafting of unusable item.

### Description

SSIA

### How to test

1. Go to workshop and try to combine unusable item.
2. Check if confirm popup is shown.

### Related Links
https://app.asana.com/0/958521740385861/1201776652036375

### Required Reviewers
@planetarium/9c-dev @Namyujeong
